presto-main/src/test/java/io/prestosql/sql/planner/TestQuantifiedComparison.java

```diff
@@ -35,7 +37,12 @@
     public void testQuantifiedComparisonEqualsAny()
     {
         String query = "SELECT orderkey, custkey FROM orders WHERE orderkey = ANY (VALUES ROW(CAST(5 as BIGINT)), ROW(CAST(3 as BIGINT)))";
-        assertPlan(query, anyTree(
+        assertPlan(
+                query,
+                Session.builder(getQueryRunner().getDefaultSession())
```

Comment:
Do we care that it's not rewritten? In other tests it seem to make sense, but in this one not neccecerly.